### ISSUE TITLE
[FR] Custom sleep timer minutes

### ISSUE THREAD

**Issue by 0verEngineer** [GrakovNe/lissen-android#69]
It would be very nice to set a custom amount if minutes for the sleep timer.

**Comment by GrakovNe** [comment]
Yes, it's planned for the nearest future!

Thanks a lot

**Comment by GrakovNe** [comment]
UPD: i'm still trying to get clear UI/UX for this feature. If someone could suggest it - you're very welcome!

**Comment by GrakovNe** [comment]
> is it possible to show the current sleep timer remaining on the icon itself, when a sleep timer is set?

yes, i'm come to this few time ago. Seems like it shall be introduced. But seems like I can't provide good UX to "custom time feature"

**Comment by GrakovNe** [comment]
After six months, i think it's time to give up

**Comment by 0verEngineer** [comment]
I don't get this. Why do you let those crazy UX/UI rules you’ve imposed on yourself get in your way so much?
